catalog.html

```diff
@@ -81,9 +83,32 @@ <h1 class="inner-header-title">Моноподы для селфи</h1>
         <div class="filters-top">
           <h2 class="filters-top">фильтр:</h2>
           <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
+            <fieldset class="catalog-filter-group">
+              <legend class="catalog-filter-title">Цена</legend>
+
+              <label class="catalog-filter-label">
+                от <input class="catalog-filter-input" type=" number" name="min-price">
+              </label>
+
+              <label class="catalog-filter-label">
+                до <input class="catalog-filter-input" type=" number" name="min-price">
+              </label>
```

Comment:
```suggestion
              <label class="catalog-filter-label">
                от <input class="catalog-filter-input" type="number" name="min-price">
              </label>

              <label class="catalog-filter-label">
                до <input class="catalog-filter-input" type="number" name="min-price">
              </label>
```
Пробелы забыли убрать.